eg/Classes/MainFrame/LogCtrl.py

```diff
@@ -55,25 +73,12 @@ def __init__(self, parent):
 
         self.SetImageList(eg.Icons.gImageList, wx.IMAGE_LIST_SMALL)
 
-        sysColour = eg.colour.windowBackground
         sysTextColour = eg.colour.windowText
-        oddColour = eg.colour.GetOddLogColour()
-
-        self.attr1 = wx.ListItemAttr()
-        self.attr1.BackgroundColour = oddColour
-        self.attr1.TextColour = sysTextColour
-
-        self.attr2 = wx.ListItemAttr()
-        self.attr2.BackgroundColour = sysColour
-        self.attr2.TextColour = sysTextColour
 
-        self.attr3 = wx.ListItemAttr()
-        self.attr3.BackgroundColour = oddColour
-        self.attr3.TextColour = (255, 0, 0)
-
-        self.attr4 = wx.ListItemAttr()
-        self.attr4.BackgroundColour = sysColour
-        self.attr4.TextColour = (255, 0, 0)
+        self.attr1, self.attr2 = _create_colour_attributes(sysTextColour)
+        self.attr3, self.attr4 = _create_colour_attributes((255, 0, 0))
+        self.attr5, self.attr6 = _create_colour_attributes((89, 131, 165))
+        self.attr7, self.attr8 = _create_colour_attributes((206, 151, 22))
```

Comment:
Don't kick me 🤓 but i find the colors a bit difficult to read. What about changing the background color for warnings and tracebacks?
```
        # Normal
        self.attr1 = wx.ListItemAttr(colBack=eg.colour.GetOddLogColour())
        self.attr2 = wx.ListItemAttr()
        # Traceback
        self.attr3 = wx.ListItemAttr(colBack=(255, 160, 160))
        self.attr4 = wx.ListItemAttr(colBack=(235, 140, 140))
        # Debug
        self.attr5 = wx.ListItemAttr(
            colText=(50, 40, 190),
            colBack=eg.colour.GetOddLogColour()
        )
        self.attr6 = wx.ListItemAttr(colText=(50, 40, 190))
        # Warning
        self.attr7 = wx.ListItemAttr(colBack=(255, 235, 90))
        self.attr8 = wx.ListItemAttr(colBack=(235, 215, 70))
```
(I've also changed the debug text color a little)